packages/@glimmer/runtime/lib/dom/helper.ts

```diff
@@ -277,4 +277,4 @@ helper = domChangesSvgElementFix(doc, helper, SVG_NAMESPACE);
 export default helper;
 export const DOMTreeConstruction = DOM.DOMTreeConstruction;
 export type DOMTreeConstruction = DOM.DOMTreeConstruction;
-export { Namespace as DOMNamespace } from './interfaces';
+export { Simple.Namespace as DOMNamespace };
```

Comment:
The tests also don't like  this. Same error as bellow: 

```Error: Could not find module `@glimmer/interfaces` imported from `@glimmer/runtime/lib/dom/helper````.

Note that this error only occurs when exporting.